Services/CO.CDP.EntityVerification/Persistence/DatabasePponRepository.cs

```diff
@@ -16,6 +16,7 @@ public void Save(Ppon identifier)
         }
         catch (DbUpdateException cause)
         {
+            context.Remove(identifier);
```

Comment:
Thanks for reviewing. Basically if you don't remove it it remains part of the context, therefore if a clash occurs when it retries it will try and add the previous one that failed. I wasn't this behavior I would have thought the transaction would remove it but it doesn't.